src/hb-ot-font.cc

```diff
@@ -228,6 +228,32 @@ hb_ot_get_glyph_from_name (hb_font_t *font HB_UNUSED,
 }
 #endif
 
+#define _HB_OT_STYLE_SLANT HB_TAG ('s','l','n','t')
+#define _HB_OT_STYLE_ITALIC HB_TAG ('i','t','a','l')
+
+static float
+hb_ot_get_font_italic_angle (hb_font_t *font)
+{
+#ifndef NO_VAR
+  hb_ot_var_axis_info_t info;
+  /* Check first if is set by variation as
+     https://docs.microsoft.com/en-us/typography/opentype/spec/dvaraxistag_slnt */
+  if (hb_ot_var_find_axis_info (font->face, _HB_OT_STYLE_SLANT, &info) &&
+      info.axis_index < font->num_coords)
+    return (float) font->coords[info.axis_index] / 16384.f;
```

Comment:
Maybe subject of race condition? I wish coords be represented with hb_array_t internally.

Reply:
Changing font while using it from another thread is unsupported behavior.

Reply:
And no `hb_array_t`, nothing short of a mutex will fix that.

Reply:
> Changing font while using it from another thread is unsupported behavior.

Good to know and makes sense also

> And no hb_array_t, nothing short of a mutex will fix that.

Correct, I should've reached to that